### ISSUE TITLE
Bug in aura and other bug

### ISSUE THREAD

**Issue by SoraKohaku** [mmcc/videojs-skin-twitchy#2]
I got bug in aura. and about that button for maximize not likely good. can be it small?bar so much far away from the GUI

![bugskintwitchy](https://cloud.githubusercontent.com/assets/11433397/13721984/f45dada4-e868-11e5-8fac-94d1d630e9c8.png)

Thank you

**Comment by SoraKohaku** [comment]
![bugskintwitchy](https://cloud.githubusercontent.com/assets/11433397/13721991/506d1daa-e869-11e5-8569-cde8c07642e8.png)

PS:i think no need the bar of that. too much long way the number 2

Update I mean this. thank you :dancer:

**Comment by mmcc** [comment]
I assume by "Aura" you mean Aurora the FF beta? It looks fine for me in FF 43, so unless there was a recent regression...

With the progress bar expanded (on hover)
![FF Expanded](https://www.dropbox.com/s/6spaa14bv0r3r4a/Screenshot%202016-03-14%2010.42.57.png?raw=1)

Without: 
![FF closed](https://www.dropbox.com/s/ezl291l89oex59n/Screenshot%202016-03-14%2010.44.09.png?raw=1)

Regarding the spacing for the fullscreen button, this was meant to be a clone of the Twitch skin, so it's not really designed for a control bar full of stuff. That being said, this skin should be updated to use a width auto spacer so you can position the other elements wherever you'd like.

**Comment by mmcc** [comment]
Actually, newer versions of VJS might have broken this. We're pulling in a development version for the demo here. I'll have to look into it, but fair warning, it's not going to be this week.

**Comment by SoraKohaku** [comment]
yes I used v5.8.0. hmmm ok then I waiting. If I have time I try solve then. maybe in CSS or need put on JS. I just try later.

Okay already Put so not bug. I think the bar not sync like that so I just remove when you fixed it.

I remove in tag line

```
.vjs-skin-twitchy:hover .vjs-control.vjs-progress-control {
  height: 3em;
  top: -3em;
}
```

already change to #FF0000 still not appear. 

```
.vjs-skin-twitchy {
  color: #ff0000;
}
```

![Uploading bugskintwitchy.png…]()

**Comment by SoraKohaku** [comment]
![bugskintwitchy](https://cloud.githubusercontent.com/assets/11433397/13867807/58d9b760-ecf5-11e5-8b9c-e9a235869f3f.png)

I fixed it little ahaha. just bar not yet fixed @_@

because when mouse hover not full. In new Version VideoJS. so make it difficult little for sync with same in yours.

**Comment by mmcc** [comment]
Issues should be closed with #4
